config.sample.toml

```diff
@@ -51,7 +51,7 @@ password = "alphadose"
 [mongo]
 # For databases with authentication
 # use the following URL format `mongodb://username:password@host:port`.
-url = "mongodb://localhost:27017"
+url = "mongodb://localhost:27019"
```

Comment:
If you have populated these values
```
MONGO_INITDB_ROOT_USERNAME = "YOUR_ROOT_NAME"   # Root user of MongoDB server inside the container
MONGO_INITDB_ROOT_PASSWORD = "YOUR_ROOT_PASSWORD"   # Root password of MongoDB server inside the container
```

then the connection URL will be `mongodb://YOUR_ROOT_NAME:YOUR_ROOT_PASSWORD@localhost:27019`